src/main/java/org/pac4j/dropwizard/Pac4jFactory.java

```diff
@@ -135,6 +138,28 @@ public void setCallbackUrlResolver(
         this.callbackUrlResolver = callbackUrlResolver;
     }
 
+    /**
+     * @since 1.1.0
+     */
+    @JsonProperty
+    public Boolean getSessionEnable() {
+        return sessionEnable;
+    }
+
+    /**
+     * 
+     * @param sessionEnable
+     *            if <code>true</code> session will be enabled at Jetty level
+     *            not matter what, if <code>false</code> it won't no matter what
+     *            and if <code>null</code> then sessions will be enabled only if
+     *            there is {@link IndirectClient} in {@link #getClients()}.
```

Comment:
Yep, that's the main point I wanted to discuss. My original desire was to make things works out of the box without configuration, so I thought I would do this tri-state thing.
But maybe it would be better to have it enabled by default and that's all.